tools/run-wpt.mjs

```diff
@@ -0,0 +1,184 @@
+ * limitations under the License.
+ */
+
+import {spawnSync} from 'child_process';
+import {cpus} from 'os';
+import {dirname} from 'path';
+
+process.chdir(dirname(dirname(process.argv[1])));
+
+function log(message) {
+  // eslint-disable-next-line no-console
+  console.log(`(${process.argv[1]}) ${message}`);
+}
+
+function usage() {
+  log(
+    `Usage: [CHROMEDRIVER=<true | false>] [HEADLESS=<true | false>] [UPDATE_EXPECTATIONS=<true | false>] ${process.argv[1]} [webdriver/tests/bidi/[...]]`
+  );
+}
+
+if (
+  process.argv.length > 2 &&
+  (process.argv[2] === '-h' || process.argv[2] === '--help')
+) {
+  usage();
+  process.exit(0);
+}
+
+const isLinux = process.platform === 'linux';
+const isMacOS = process.platform === 'darwin';
+
+// The path to the browser binary.
+const BROWSER_BIN =
+  process.env.BROWSER_BIN ||
+  (isLinux
+    ? '/usr/bin/google-chrome-unstable'
+    : isMacOS
+    ? '/Applications/Google Chrome Dev.app/Contents/MacOS/Google Chrome Dev'
+    : '');
+
+// Whether to use Chromedriver with mapper.
+const CHROMEDRIVER = process.env.CHROMEDRIVER || 'false';
+
+// Whether to start the server in headless or headful mode.
+const HEADLESS = process.env.HEADLESS || 'true';
+
+// The path to the WPT manifest file.
+const MANIFEST = process.env.MANIFEST || 'MANIFEST.json';
+
+// The browser product to test.
+const PRODUCT = process.env.PRODUCT || 'chrome';
+
+// Multiplier relative to standard test timeout to use.
+const TIMEOUT_MULTIPLIER = process.env.TIMEOUT_MULTIPLIER || '8';
+
+// Whether to update the WPT expectations after running the tests.
+const UPDATE_EXPECTATIONS = process.env.UPDATE_EXPECTATIONS || 'false';
+
+// Whether to enable verbose logging.
+const VERBOSE = process.env.VERBOSE || 'false';
+
+// The path to the WPT report file.
+const WPT_REPORT = process.env.WPT_REPORT || 'wptreport.json';
+
+// Only set WPT_METADATA if it's not already set.
+let WPT_METADATA;
+if (typeof process.env.WPT_METADATA === 'undefined') {
+  if (CHROMEDRIVER === 'true') {
+    WPT_METADATA = 'wpt-metadata/chromedriver/headless';
+  } else {
+    WPT_METADATA =
+      HEADLESS === 'true'
+        ? 'wpt-metadata/mapper/headless'
+        : 'wpt-metadata/mapper/headful';
+  }
+} else {
+  WPT_METADATA = process.env.WPT_METADATA;
+}
+
+if (HEADLESS === 'true') {
+  log('Running WPT in headless mode...');
+} else {
+  log('Running WPT in headful mode...');
+}
+
+const wptRunArgs = [
+  '--binary',
+  BROWSER_BIN,
+  '--webdriver-binary',
+  'tools/run-bidi-server.mjs',
+  `--webdriver-arg=--headless=${HEADLESS}`,
+  '--log-wptreport',
+  WPT_REPORT,
+  '--manifest',
+  MANIFEST,
+  '--metadata',
+  WPT_METADATA,
+  '--no-manifest-download',
+  '--skip-implementation-status',
+  'backlog',
+  '--timeout-multiplier',
+  TIMEOUT_MULTIPLIER,
+];
+
+if (VERBOSE === 'true') {
+  wptRunArgs.push(
+    '--debug-test',
+    '--log-mach',
+    '-',
+    '--log-mach-level',
+    'info'
+  );
+} else if (HEADLESS === 'true') {
```

Comment:
Please update to use the part from this PR https://github.com/GoogleChromeLabs/chromium-bidi/pull/1025